app/src/androidTest/java/swati4star/createpdf/util/BrushUtilsTest.java

```diff
@@ -0,0 +1,31 @@
+package swati4star.createpdf.util;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import swati4star.createpdf.R;
+import swati4star.createpdf.model.BrushItem;
+import static org.junit.Assert.assertEquals;
+
+public class BrushUtilsTest {
+
+    ArrayList<BrushItem> defaultBrushItems = new ArrayList<>();
+
+    @Before
+    public void setUp() throws Exception {
+        defaultBrushItems.add(new BrushItem(R.color.mb_white));
```

Comment:
one suggestion.
 - Can you add this setup or initialize function in BrushUtils class only? (call this function in getInstance method)
 - Make a private member mBrushItems in BrushUtils class and just return it in getBrushItems function
- Optional: you can also add `addBrushItems` method in `BrushUtils` and add a testcase for that too

Reason for this is,
If we somehow change the default items in future. We'll have to change those here.